### ISSUE TITLE
Allow for different DFT grid sizes in qcxms.in  [improvement]

### ISSUE THREAD

**Issue by tobigithub** [qcxms/QCxMS#18]
Hi,
for DFT calculations with QCxMS it would be nice to allow for different grid settings in the qcxms.in settings file.

Reason, the DFT grid size controls accuracy, but also speed. 
So for speedier calculations one might try a coarse DFT grid size first, which is fast, but also leads to larger errors.
For more accurate calculations, one could select a finer DFT grid size, which are more accurate, but also significantly slower.

Relevant: https://cen.acs.org/physical-chemistry/computational-chemistry/Density-functional-theory-error-discovered/97/web/2019/07

Ref: **Popular Integration Grids Can Result in Large Errors in DFT-Computed Free Energies**
https://doi.org/10.26434/chemrxiv.8864204.v5

For TurboMole, the settings are set in the control file gridsize 1..6 and gridsize m3..m5 in (QCxMS/src/tm.f90)
https://github.com/qcxms/QCxMS/search?q=gridsize+m4
````
$dft
 functional pbe0
 gridsize m4
````
with qcxms.in I used the following settings:
````
tmol
pbe0
def2-SVP
cid
elab 40
maxcoll 3
noesi
````

For ORCA this would be 
````
! defgrid1
! defgrid2 (default).
! defgrid3
````

Tobias

**Comment by JayTheDog** [comment]
For ORCA the new grid options are implemented and some version control is provided. Use `orca4` for versions 4.0 and higher or `orca5` for the new version ORCA 5.0. [default: `orca5`]. 

However, in the `tm.f90` file, there is the following comment:
` !     now the setup: hybrid func. b3lyp, grid m4 (savings with m3 are marginal
  !     and m3 grid with fermi smearing produces a noisy gradient!). Real savings
  !     come with rij, extol 2.500, and scfonv 6. CAB 6.10.15
  !     gridsize m3 for GGAs - tests June 2016 CAB `

i.e. that the gridsizes were checked and it was found that the speed-up does not come from grid sizes.

**Comment by JayTheDog** [comment]
Fixed in version 5.1.3 for ORCA. For Turbomole this is not needed.
